lib/init.coffee

```diff
@@ -1,8 +1,12 @@
+{CompositeDisposable} = require 'atom'
+{findFile, exec, tempFile} = helpers = require 'atom-linter'
+
 module.exports =
   config:
-    gjslintExecutablePath:
+    executablePath:
       type: 'string'
-      default: ''
+      title: 'gjslint executable path'
+      default: '/usr/local/bin/gjslint'
```

Comment:
It might be a better idea to just have the default be `gjslint`, that way if it's in the user's path anywhere it should be found automatically, with them having the option to specify the full path if necessary.